pennylane_lightning/lightning_qubit.py

```diff
@@ -867,16 +975,24 @@ def var(self, observable, shot_range=None, bin_size=None):
             samples = self.sample(observable, shot_range=shot_range, bin_size=bin_size)
             return np.squeeze(np.var(samples, axis=0))
 
-        # Initialization of state
-        ket = np.ravel(self._pre_rotated_state)
+        batched_var = []
```

Comment:
Maybe out of scope for this PR, and mostly out of curiosity: Why does `var` not require the distinction between `SparseHamiltonian`, `Hamiltonian/Hermitian` and others? :)

Reply:
Neither lightning nor default qubits support the calculation of var for Hamiltonians.

Reply:
...fair point :laughing: